Refactor the Query DSL section (#2904)
opensearch-project/documentation-website: _opensearch/query-dsl/text-analyzers.md

layout: default
title: Text analyzers
parent: Query DSL
nav_order: 75

> ---

Reviewer: Can we make these bullet points instead of a numbered list?

OpenSearch supports the following text analyzers:

> - **Standard analyzer** – Parses strings into terms at word boundaries per the Unicode text segmentation algorithm. It removes most, but not all, punctuation. It converts strings to lowercase. You can remove stop words if you turn on that option, but it does not remove stop words by default.

Reviewer 2: ```suggestion
- **Standard analyzer** – Parses strings into terms at word boundaries according to the Unicode text segmentation algorithm. It removes most, but not all, punctuation and converts strings to lowercase. You can remove stop words if you enable that option, but it does not remove stop words by default.
```

- **Simple analyzer** – Converts strings to lowercase and removes non-letter characters when it splits a string into tokens on any non-letter character.
- **Whitespace analyzer** – Parses strings into terms between each whitespace.

> - **Stop analyzer** – Converts strings to lowercase and removes non-letter characters by splitting strings into tokens at each non-letter character. It also removes stop words (e.g., "but" or "this") from strings.

Reviewer 2: ```suggestion
- **Stop analyzer** – Converts strings to lowercase and removes non-letter characters by splitting strings into tokens at each non-letter character. It also removes stop words (for example, "but" or "this") from strings.
```
